<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1000</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mineral Water Distribution System (CP Project by Sauban, Shehryar &amp; Salman)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1001</width>
      <height>701</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_Customers">
     <attribute name="title">
      <string>Customers</string>
     </attribute>
     <widget class="QPushButton" name="btn_ShowSummary">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>630</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Show Report</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_AddCustomer">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Customer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_RemoveCustomer">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>630</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Customer</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidgetCustomers">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>971</width>
        <height>611</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Employees">
     <attribute name="title">
      <string>Employees</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetEmployees">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>971</width>
        <height>521</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_AddEmployee">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Employee</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_RemoveEmployee">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Employee</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxEmployee">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lbl1">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Address :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>20</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl3">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Role :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl4">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Salary :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblSalary">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>80</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ag</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAddress">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>0</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ad</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblPhone">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ph</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblRole">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>60</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>rl</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAge">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>sl</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl5">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Age :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_RemoveEmployeeRecord">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>630</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Record</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_AddEmployeeRecord">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Record</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Transport">
     <attribute name="title">
      <string>Transport</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidgetTransport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>971</width>
        <height>521</height>
       </rect>
      </property>
      <property name="columnCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxTransport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_AddTransport">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Transport</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_RemoveTransport">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Transport</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl11">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reg. No.:</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblAreaCode">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ac</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl22">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Area Code :</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblRegistrationNumber">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>10</y>
        <width>631</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>rg</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_AddTransportRecord">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>630</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Record</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_RemoveTransportRecord">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>630</y>
        <width>110</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Remove Record</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
